tests/apollo/test_skvbc.py

```diff
@@ -65,7 +65,7 @@ async def test_request_missing_data_from_previous_window(self, bft_network, exch
         """
 
         if exchange_keys:
-            await bft_network.check_initital_key_exchange(full_key_exchange=False)
+            await bft_network.check_initial_key_exchange(full_key_exchange=False)
```

Comment:
Are the changes in this file related to the sanitizer & heaptrack work, or are they "while I'm in here" changes.  If they're not related to sanitizer and heaptrack work, I think they should be a separate delta.

Reply:
@samcramer , I assume that you are watching a full diff.
I did put it in a seperate commit.
If you want to watch diff bt commit, enter Pr from https://github.com/vmware/concord-bft/pull/2877/commits